src/Microsoft.TestPlatform.TestHostProvider/Hosting/TestHostManagerCallbacks.cs

```diff
@@ -59,8 +59,11 @@ public static void ErrorReceivedCallback(StringBuilder testHostProcessStdError,
 
             if (exitCode != 0)
             {
-                EqtTrace.Error("Test host exited with error: {0}", testHostProcessStdErrorStr);
-                messageLogger.SendMessage(TestMessageLevel.Error, testHostProcessStdErrorStr);
+                EqtTrace.Error("Test host exited with error: '{0}'", testHostProcessStdErrorStr);
```

Comment:
We can probably remove this. It is resulting in output like the following, note the duplication.

```
Microsoft (R) Test Execution Command Line Tool Version 15.3.0-dev
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...
[xUnit.net 00:00:00.5125931]   Discovering: trial
[xUnit.net 00:00:00.6323365]   Discovered:  trial
[xUnit.net 00:00:00.6750969]   Starting:    trial
Unhandled Exception: System.Diagnostics.Debug+DebugAssertException: Oops
   at System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)
   at System.Environment.get_StackTrace()
   at System.Diagnostics.Debug.Assert(Boolean condition, String message, String detailMessage)
   at System.Diagnostics.Debug.Assert(Boolean condition, String message)
   at trial.UnitTest1.<>c.<Fails>b__1_0() in G:\tmp\trial\UnitTest1.cs:line 19
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Diagnostics.Debug.WindowsDebugLogger.ShowAssertDialog(String stackTrace, String message, String detailMessage)
   at System.Diagnostics.Debug.Assert(Boolean condition, String message, String detailMessage)
   at System.Diagnostics.Debug.Assert(Boolean condition, String message)
   at trial.UnitTest1.<>c.<Fails>b__1_0() in G:\tmp\trial\UnitTest1.cs:line 19
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)

The active test run was aborted. Reason: Unhandled Exception: System.Diagnostics.Debug+DebugAssertException: Oops
   at System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)
   at System.Environment.get_StackTrace()
   at System.Diagnostics.Debug.Assert(Boolean condition, String message, String detailMessage)
   at System.Diagnostics.Debug.Assert(Boolean condition, String message)
   at trial.UnitTest1.<>c.<Fails>b__1_0() in G:\tmp\trial\UnitTest1.cs:line 19
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Diagnostics.Debug.WindowsDebugLogger.ShowAssertDialog(String stackTrace, String message, String detailMessage)
   at System.Diagnostics.Debug.Assert(Boolean condition, String message, String detailMessage)
   at System.Diagnostics.Debug.Assert(Boolean condition, String message)
   at trial.UnitTest1.<>c.<Fails>b__1_0() in G:\tmp\trial\UnitTest1.cs:line 19
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
```